spec/expand/dollar_sign/dollar_operator.spec

```diff
@@ -0,0 +1,8 @@
+# outside of a cluster definition the dollar sign is not a metacharacter
+$foo
+$foo
```

Comment:
Generally, non-metacharacters are literals. I don't *think* I'd oppose it being an error, as long as it still does the right thing inside cluster definitions. Mostly concerned about the syntax actually doing something it shouldn't -- right now it seems to be an alias for `@`.